ansible/roles/openshift_setup/tasks/main.yml

```diff
@@ -128,15 +127,6 @@
       - "{{ docker_images_group1 }}"
       - "{{ docker_images_group2 }}"
 
-  - name: Remove {{ oc_host_config_dir }}/master/master-config.yaml when resetting cluster
```

Comment:
I think I was confused by [this comment](https://github.com/fusor/catasb/blob/dev/ansible/roles/openshift_setup/tasks/main.yml#L138-L139). I thought that the intent of the reset was to avoid the extra up/down to generate the config.  If we want to generate a new config every time, should we just do the extra up/down any time we use a custom config?